client/app/telecom/overTheBox/details/overTheBox-details.controller.js

```diff
@@ -382,7 +376,7 @@ angular.module("managerApp").controller("OverTheBoxDetailsCtrl", function ($scop
             function (devices) {
                 self.device = devices;
                 self.kpiInterfaces = devices.networkInterfaces.filter(function (netInterface) {
-                    return /^if\d+$/i.test(netInterface.name) || /^tun\d+$/.test(netInterface.name);
+                    return netInterface.gateway !== null;
```

Comment:
If it's `undefined` it won't be filtered because of strict equal. I think it's better to `return !netInterface.gateway;`